src/fairseq2/recipes/lm/_online_finetune/_online_dpo.py

```diff
@@ -160,6 +159,8 @@ def validate_reward(self, prompt_batch: PromptBatch) -> tuple[Tensor, int]:
         if self._gangs.dp.rank == 0:
             policy_sampling_params = copy(self._vllm_model.sampling_params)
             policy_sampling_params.n = 1
+            for k, v in self._loss_config.validation_vllm_sampling_params.items():
+                policy_sampling_params.__setattr__(k, v)
```

Comment:
this will overwrite the `policy_sampling_params.n = 1` if `n` is one of the `validation_vllm_sampling_params`